test/Land/Model/test_heat_parameterizations.jl

```diff
@@ -0,0 +1,73 @@
+using MPI
+using OrderedCollections
+using StaticArrays
+using Test
+
+using CLIMAParameters
+using CLIMAParameters.Planet: ρ_cloud_liq, ρ_cloud_ice, cp_l, cp_i, T_0, LH_f0
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+using ClimateMachine
+using ClimateMachine.Land.SoilHeatParameterizations
+
+@testset "Land heat parameterizations" begin
+    FT = Float64
+    # Density of liquid water (kg/m``^3``)
+    _ρ_l = FT(ρ_cloud_liq(param_set))
+    # Density of ice water (kg/m``^3``)
+    _ρ_i = FT(ρ_cloud_ice(param_set))
+    # Volum. isoboric heat capacity liquid water (J/m3/K)
+    _cp_l = FT(cp_l(param_set) * _ρ_l)
+    # Volumetric isoboric heat capacity ice (J/m3/K)
+    _cp_i = FT(cp_i(param_set) * _ρ_i)
+    # Reference temperature (K)
+    _T_ref = FT(T_0(param_set))
+    # Latent heat of fusion at ``T_0`` (J/kg)
+    _LH_f0 = FT(LH_f0(param_set))
+
+    @test round(
```

Comment:
im curious why you decided to use round here? Im not not sure that what I did is any better, but on the LHS I evaluated the function explicitly, e.g.
@test volumetric_heat_capacity(0.25, 0.05, 1e6, _cp_l, _cp_i) == 1e6 + 0.25 * _cp_l + 0.05 * _cp_i.

Charlie also mentioned at one point testing on an array of values, not just a single value, though this is  a fairly simple function and you tested it at "interesting" values :) (i.e. not at 0)

Reply:
ah! i can change it to this, then we know where the values are coming from (calculated them on matlab). I thought at first it would be easier to type in the answer directly, and round to the number of sig figs that the function gives out